### PR TITLE
Updates to resource estimation output

### PR DESCRIPTION
**Pull request by msoeken**
This also adds a configuration setting `estimator.summary` to enable and disable a summarized view (default is a detailed view).

### CODE REVIEW

File: src/AzureClient/Visualization/ResourceEstimationEncoder.cs [hunk 1]
```diff
@@ -92,27 +97,50 @@ public record class ResourceEstimationToHtmlEncoder(ILogger? Logger = null) : IR
                         }
 
                         .aqre-tooltip:hover .aqre-tooltiptext {
-                            visibility: visible;
+                          visibility: visible;
```
Comment (kuzminrobin) [inline src/AzureClient/Visualization/ResourceEstimationEncoder.cs:100]: I have an impression that this indentation change was not planned (based on indentation at green L85-).

**Review by kuzminrobin — APPROVED** [review]
Approving to unblock. But I would recommend at least one more approval from those familiar with this code base.

File: src/AzureClient/Visualization/ResourceEstimationEncoder.cs (hunk 1 above)
Comment (msoeken) [reply src/AzureClient/Visualization/ResourceEstimationEncoder.cs]: Fixed.

File: src/Jupyter/ConfigurationSource/IConfigurationSource.cs [hunk 2]
```diff
@@ -246,6 +246,14 @@ public T GetOptionOrDefault<T>(string optionName, T defaultValue, Func<string, T
         public string WorkspaceName =>
             GetOptionOrDefault("azure.quantum.workspace.name", string.Empty);
 
+        /// <summary>
+        ///     Whether the resource estimation table should be shown in summary mode or not.
+        ///     In summary mode the explanations are shown as a tooltip, otherwise, they are inlined
+        ///     into the table cells.  The default value is `false`.
+        /// </summary>
+        public bool EstimatorSummary =>
+            GetOptionOrDefault("estimator.summary", false);
```
Comment (anpaz) [inline src/Jupyter/ConfigurationSource/IConfigurationSource.cs:255]: I would name this `estimator.showSummary`, since it's a bool flag.

**Review by anpaz — APPROVED** [review]

File: src/Jupyter/ConfigurationSource/IConfigurationSource.cs (hunk 2 above)
Comment (msoeken) [reply src/Jupyter/ConfigurationSource/IConfigurationSource.cs]: Changed that.

**Pull request merged**